### ISSUE TITLE
Add guidance for not using "prefix" in JSON-LD

### ISSUE THREAD

**Issue by OR13** [w3c-ccg/security-vocab#57]
@dlongley I heard you prefer not to register prefixes, and instead use full URLs... 

That should be documented in this repo somewhere... I am attempting to follow that guidance, and I will reject PRs that fail to do so from now on.

**Comment by dlongley** [comment]
Yes, prefixes generally cause trouble. They can be "ok" if appropriately scoped and well thought out, but that's asking too much of authors most of the time, so it's better to just avoid them. Most of the trouble is from the fact that JSON-LD does not have a mechanism for making prefixes only meaningful *inside* `@context`. So they aren't merely shortcuts for term definitions, rather, they can spill out into the data, which can lead to undesirable or unexpected behavior.
